### DISCUSSION TITLE
Prevent apps from stealing focus while opening

### DISCUSSION THREAD

**Discussion by grgwsmth** [ttscoff/bunch-docs#291] [Q&A] [answered]
Seems like this is extremely difficult. I've tried several different Apple Scripts to prevent apps from stealing focus while opening, with no success. I'd love to have the ability to prevent all apps from EVER taking focus, unless I decide to focus the app. But is there at least a way to prevent all apps from taking focus while opening, with Bunch?

**Comment by ttscoff** [comment, marked as answer]
Not really. macOS offers the ability to launch an app in the background, 
and Bunch usually takes advantage of it automatically, but the only way 
to force an app to background is to follow it with an underscore in 
Bunch. There's no "Launch all in background" command available.

-Brett

**Reply by grgwsmth** [reply]
Thank you! And thanks for creating such a cool app 😃
